quantum/via.c

```diff
@@ -397,6 +397,12 @@ void raw_hid_receive(uint8_t *data, uint8_t length) {
             dynamic_keymap_set_buffer(offset, size, &command_data[3]);
             break;
         }
+#ifdef RAW_HID_CMD
+        case RAW_HID_CMD: {
```

Comment:
This doesn't work, and will error during compiling, if `RAW_HID_CMD` is defined.  

At best, this should just simply replace the default case, if anything

Reply:
Need to define the command value such as
#define RAW_HID_CMD 0xA0
or
#define RAW_HID_CMD 0xA0 ... 0xAA

Reply:
I think `RAW_HID_CMD > id_dynamic_keymap_set_buffer` is redundancy here, compiler will through error is there are two same case IDs. 
As a developer, I prefer to know there is error ealier, if there via IDs extend and there is confiliction, we can get the error first time, this is the pros of putting new RAW_HID_CMD here. 
Putting RAW_HID_CMD  to `default` hanlder would make it easy to maintenance, since there won't be compiling error, and no one will know this is confict if via IDs extend until someone test it, especially when the conficted IDs are seldom used. 

Anyway I moved it to `default` handler as suggestion, experienced programer always keeps distance to last ID of system.